src/coreclr/jit/lower.cpp

```diff
@@ -2783,6 +2783,36 @@ GenTree* Lowering::LowerCompare(GenTree* cmp)
     }
 #endif
 
+    // Optimizes (X & 1) == 1 to (X & 1)
+    //
+    //                        EQ/NE                   AND
+    //                        /  \                   /  \.
+    //                      AND   CNS 1  ->         x   CNS 1
+    //                     /   \.
+    //                    x   CNS 1
```

Comment:
The picture should go; it is not accurate for LIR (where trees aren't ordered), and the `Optimizes (X & 1) == 1 to (X & 1)` comment should be enough.